<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WmConfig_UI</class>
 <widget class="QWidget" name="WmConfig_UI">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="kwinRB">
     <property name="text">
      <string>&amp;Use the default KDE window manager (KWin)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="differentRB">
     <property name="text">
      <string>Use a different &amp;window manager:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KComboBox" name="wmCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="configureButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Note: Most window managers have their own configuration and do not follow KDE settings.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
   <customwidget>
     <class>KComboBox</class>
     <extends>QComboBox</extends>
     <header location="global">KDE/KComboBox</header>
   </customwidget>
   <customwidget>
     <class>KPushButton</class>
     <extends>QPushButton</extends>
     <header location="global">KDE/KPushButton</header>
   </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>differentRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>wmCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
